example-scripts/gstreamer/ingest.py

```diff
@@ -203,8 +219,8 @@ def run_pipeline(pipeline, args):
     clock = GstNet.NetClientClock.new('voctocore', core_ip, 9998, 0)
     print('obtained NetClientClock from host', clock)
 
-    print('waiting for NetClientClock to sync…')
-    clock.wait_for_sync(Gst.CLOCK_TIME_NONE)
+    print('waiting for NetClientClock to sync')
+    #clock.wait_for_sync(Gst.CLOCK_TIME_NONE)
```

Comment:
This is a blocker.  

Using core's clock remotely is one of the main features of injest.py.

Reply:
I understand the importance of using core's clock.   I was hoping for some help figuring out the cause of the failures I've been having with it.

I will keep trying - I fear it may be that gstreamer python bindings on windows just don't support the clock.

Reply:
one could make it optional and default to False on windows.